packages/simple-hub/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "module": "commonjs",
     "target": "es2017",
     "noImplicitAny": false,
+    "rootDir": ".",
```

Comment:
What made this necessary? Just curious

Reply:
Typescript compilation started failing for me locally (again...). And I had to add this to get rid of the errors for importing `env` module.